.travis.yml

```diff
@@ -1,6 +1,14 @@
 language: cpp
 sudo: required
 
+os:
+  - linux
+  - osx
+
+compiler:
+  - gcc
+  - clang
```

Comment:
@OXINARF Please go ahead, if I am honest I am finding this quite challenging :) . I will keep plugging away but by all means make a new PR if it solves a problem